logfilter/logfilter.go

```diff
@@ -162,6 +164,19 @@ func containsBlacklistedString(message string, blacklistedStrings []string) bool
 	return false
 }
 
+var apiKeyQueryParamRegExp = regexp.MustCompile("apiKey=[a-zA-Z0-9]+")
```

Comment:
I suggest `api_?key=[^\s&]+` as an alternative, used in case-insensitive mode. This will probably redact more cases.